x-pack/legacy/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -18,6 +18,15 @@ import { TabNavigationProps } from './types';
 const TabContainer = styled.div`
   .euiLink {
     color: inherit !important;
+
+    &:focus {
```

Comment:
Looking deeper into this, styling is not the issue. `EuiTabs` with prop `display="condensed"` already handles these styles. 

The problem is that sometime after the original placement of the `EuiTabs` in the SIEM app, someone has added a wrapping `a.euiLink` element around the `button` elements for each tab. This is both invalid HTML and it is what appears to be causing the wonky focus state in Firefox. Rather than changing the styles, I believe we should address the invalid link wrapping the button (which will correct the style issues as well).

@angorayc, @stephmilovic: Do either of you know who might have done this? They might be able to give us some incite as to why before we change it.

Reply:
@XavierM and @patrykkopycinski had a chat about this, appears that wrapping it with a euiLink is to fix the behaviour for Firefox and IE11, therefore fixing it with styles makes sense to me.

Reply:
Yes, we need to wrap it under EuiLink, or you can not use `right click` on IE11 (for sure) and Safari (I think) to `open link in a new tab`. That's why we did like that.

Reply:
because IE11 does not see an anchor and won't href to the new route